### DISCUSSION TITLE
Possible Contribution

### DISCUSSION THREAD

**Discussion by Pulkith** [sensein/science4all#3] [General]
Hey!

Prof. Ghosh asked me to make a discussion post here about my interests, so that I may possibly be able to contribute.

I'm quite interested in Parkinson's Disease(and other neurodegenerative diseases, but I haven't really explore any of the others), neuroscience, and Machine Learning. For context, for the past few years I've been working on a method of remote(via a website) diagnosis of Parkinson's through Machine Learning. I would be interested in anything related to Machine Learning, and any sort of diagnosis software, or anything related to neuroscience. 

I usually can gain substantial of interest in most things, so if you have any projects I can work on, or if you need help with anything, I can definitely help!

I have a decent amount of experience with Machine Learning(mostly classifiers, Deep Learning, and both Supervised/Unsupervised Learning), Research(few pending publications and research at a few universities), programming languages(C++,R, Python,TF & Keras), and Neuroscience!

**Comment by satra** [comment]
@DespicableMonkey - we are really sorry that no one followed up on this. I'm going to close this issue, but if you are still interested please reopen.
